automated/linux/bootrr/bootrr.yaml

```diff
@@ -0,0 +1,32 @@
+metadata:
+    format: Lava-Test Test Definition 1.0
+    name: bootrr
+    description: "Run bootrr https://github.com/andersson/bootrr"
+    maintainer:
+        - milosz.wasilewski@linaro.org
+        - chase.qi@linaro.org
+    os:
+        - debian
+        - ubuntu
+        - centos
+        - fedora
+        - openembedded
+    scope:
+        - functional
+    devices:
+        - apq8016-sbc
+        - dragonboard-410c
+        - dragonboard-820c
+
+params:
+    # Refer to https://github.com/andersson/bootrr/tree/master/boards for boards supported.
+    BOARD: "arrow,apq8096-db820c"
+    SKIP_INSTALL: false
+
+run:
+    steps:
+        - . ./automated/lib/sh-test-lib
+        - install_deps git "${SKIP_INSTALL}"
```

Comment:
With the current approach, install_deps step can be skipped, but the following 'git clone' is required. Checking if git installed is redundant as lava run the 'run steps' with sh '-e' enabled by default. We also can do:

Add a param for existing bootrr(built-in or overlay applied somehow), when the param defined, skip both install_deps and git clone, then use the existing bootrr directory directly.

Reply:
I am merging this PR. @alimon will send another PR to add a param for using existing bootrr.